pypdf/generic/_utils.py

```diff
@@ -104,7 +104,7 @@ def read_string_from_stream(
                     # line break was escaped:
                     tok = b""
                 else:
-                    msg = rf"Unexpected escaped string: {tok.decode('utf8')}"
+                    msg = rf"Unexpected escaped string: {tok.decode('utf8','ignore')}"
```

Comment:
```suggestion
                    msg = f"Unexpected escaped string: {tok.decode('utf-8','ignore')}"
```
this is would be more consistent with the rest of the code